controllers/console_controller.go

```diff
@@ -0,0 +1,135 @@
+package controllers
```

Comment:
Question: Did you created a this file via operator-sdk scafolding?
This file should also have https://github.com/red-hat-storage/odf-operator/blob/main/hack/boilerplate.go.txt